src/utils/line.ts

```diff
@@ -430,3 +426,57 @@ const _createTokenView = (tokenResponse: TokenFragRepsonse, amount?: BigNumber,
     description: '',
   };
 };
+
+export const formatGetBorrowerQuery = (data: AggregatedCreditLine[], borrowerAddress: Address) => {
+  let aggregate: CreditLinePage = {
+    borrower: borrowerAddress,
+    principal: '0', //done
+    deposit: '0', //done
+    highestApy: ['0', '0', '0'], //todo
+    positions: [], //done
+    escrow: {
+      id: '',
+      cratio: '',
+      minCRatio: '',
+      collateralValue: '',
+      //@ts-ignore
+      deposits: [],
+    }, //todo
+    spigot: {
+      id: '',
+      tokenRevenue: {},
+    }, //todo
+
+    //CreditLinePage data
+    interest: '0', //done
+    totalInterestRepaid: '0', //to review
+    collateralEvents: [], //todo
+    creditEvents: [], //done
+  };
+
+  if (data) {
```

Comment:
these whole thing seems unecesseary or atleast sloppy and not DRY. plz refactor.